resources/lib/ui/http.py

```diff
@@ -1,6 +1,7 @@
 from http_imports import *
+from resources.lib.token_decoder import TokenDecoder
 
-_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.95 Safari/537.36'
+_USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36'
```

Comment:
with the macintosh useragent, the request didnt get through despite the reqkey cookie